contracts/btc-staking/src/state/delegations.rs

```diff
@@ -0,0 +1,102 @@
+use crate::state::points_alignment::PointsAlignment;
+use cosmwasm_schema::cw_serde;
+use cosmwasm_std::{CanonicalAddr, Order, StdResult, Storage, Uint128};
+use cw_storage_plus::{Index, IndexList, IndexedMap, KeyDeserialize, MultiIndex};
+
+/// Single delegation related information - entry per `(staking hash, finality provider public key)`
+/// pair, including distribution alignment
+#[cw_serde]
+pub struct Delegation {
```

Comment:
This is a bit confusing with the `BtcDelegation` struct. If this struct is dedicated for the reward distribution shall we rename it to something explicitly?

Reply:
Yes, we can rename to `DelegationDistribution` or `DelegationAlignment` or so.

Reply:
Done. We are creating this new struct as it's lighter than the `BtcDelegation`, and relatively independent. We can decide to merge all into one, if we want.